### PR TITLE
feat: re-organized iedc-crafts + few site layout changes

### PR DESCRIPTION
**Pull request by shaheenfazim**
+ re-organized  crafts site path.
+ redesigned crafts.
+ added product courosel.
+ added new changes to modal.
+ re-organized iedc  portfolio.

### CODE REVIEW

File: crafts/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+body {
```
Comment (nandgator) [inline crafts/css/styles.css:1]: I was thinking of such a directory structure.

```
| -- css
| -- index.html
| -- pages
    | -- crafts
        | -- css
        | -- js
        | -- crafts.html
```

I think it's good to have a structure like this, we can have folders called `alumni/` under `pages/` as and when required.

Comment (shaheenfazim) [reply crafts/css/styles.css:1]: alright, I will do this in the next push.

File: index.html [hunk 2]
```diff
@@ -207,54 +207,12 @@ <h3 class="section-subheading text-muted">Library of awesome works by IEDC</h3>
                             <img class="img-fluid" src="assets/img/portfolio/03-thumbnail.jpg" alt="..." />
                         </a>
                         <div class="portfolio-caption">
-                            <div class="portfolio-caption-heading">Finish</div>
-                            <div class="portfolio-caption-subheading text-muted">MVPs</div>
+                            <div class="portfolio-caption-heading">IEDC Summit</div>
```
Comment (nandgator) [inline index.html:210]: image

A dangling `<li>` tag, in the popup, pretty easy fix..

File: crafts/css/styles.css (hunk 1 above)
Comment (shaheenfazim) [reply crafts/css/styles.css:1]: Is it okay for now to approve I have implemented other changes to the sites too

File: index.html (hunk 2 above)
Comment (shaheenfazim) [reply index.html:210]: oh is didn't see that there my bad

Comment (shaheenfazim) [reply index.html:210]: u can close this, I could do another request.

**Comment by nandgator** [comment]
no, wait..  hold on.

**Comment by nandgator** [comment]
I'll suggest all the changes.. then merge it, finally you can make the change and create another pull

**Comment by shaheenfazim** [comment]
wait can u dm me on whatsapp

File: crafts/crafts.html [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+                    </div>
+        
+                </div>
+              </div>
+        
+        
+              <div class="col">
+                <div class="card h-100 rounded shadow-sm border-0">
+        
+                  <div class="card-body p-5"><i class="fa fa-user-circle-o fa-2x mb-3 text-info"></i>
```
Comment (nandgator) [inline crafts/crafts.html:128]: image

This fA icon is for pro I guess..

Comment (shaheenfazim) [reply crafts/crafts.html:128]: yeah I just kept that hangin i will update the icon on next push

**Comment by nandgator** [comment]
> wait can u dm me on whatsapp

I'm not using it for some time (exams).. you can hit me up on signal / telegram

**Comment by shaheenfazim** [comment]
https://meet.google.com/jqs-uroc-ryo

join

**Comment by nandgator** [comment]
> https://meet.google.com/jqs-uroc-ryo

Sorry, no time for that bro.. will see after Monday

**Comment by shaheenfazim** [comment]
ok, so i will close this and do another push

**Comment by nandgator** [comment]
I don't think there's need for closing this issue.. it's fine..

**Comment by shaheenfazim** [comment]
cause thats only makes sense to me , i dont know how to manage the merge thing so

**Comment by nandgator** [comment]
Okay, no problem them.

**Comment by shaheenfazim** [comment]
ok

**Comment by shaheenfazim** [comment]
i will request another one better

**Comment by nandgator** [comment]
By the way I'll send you a link and a photo (via telegram) which helps to understand git, do share it with the team. please.

**Comment by nandgator** [comment]
> i will request another one better

It's okay, you did an awesome job bro 👍🏻, nobody's perfect.

**Pull request closed without merging**